<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bag</class>
 <widget class="QDialog" name="Bag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>728</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>728</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>无涯</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Menu.qrc">
    <normaloff>:/icon/Resources/无涯.png</normaloff>:/icon/Resources/无涯.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/other/Resources/baofu.png);</string>
  </property>
  <widget class="QPushButton" name="qinglongjian">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>-50</y>
     <width>281</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/qinglongjian.png);
font: 26pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>青龙剑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zhengyangjian">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>531</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/zhengyangjian.png);
font: 26pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正阳剑</string>
   </property>
  </widget>
  <widget class="QLabel" name="drug12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/danyao2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drug22">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>320</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/xijiao2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="drug32">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/qinglonglin2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnzhuqueguan">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/shuimo1.png);
color: rgb(255, 255, 255);
font: 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>朱雀冠</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btndrug3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>470</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 0, 0);
font: 14pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>青龙鳞</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btndrug2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>470</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 0, 0);
font: 14pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>巨兕角</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btndrug1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/边框2.png);
color: rgb(255, 0, 0);
font: 14pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>伸腿瞪眼丸</string>
   </property>
  </widget>
  <widget class="QLabel" name="lablemoney">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/边框2.png);
font: 28pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>  银钱:</string>
   </property>
  </widget>
  <widget class="QLabel" name="money_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>650</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/透明背景.png);
font: 28pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnzhengyangguan">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/shuimo1.png);
color: rgb(255, 255, 255);
font: 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>正阳冠</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnbaihuxue">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>120</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/shuimo1.png);
color: rgb(255, 255, 255);
font: 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>白虎靴</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnzhengyangxue">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>120</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/shuimo1.png);
color: rgb(255, 255, 255);
font: 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>正阳靴</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnxuanwujia">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/shuimo1.png);
color: rgb(255, 255, 255);
font: 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>玄武重甲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnhuxinjing">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/shuimo1.png);
color: rgb(255, 255, 255);
font: 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>护心镜</string>
   </property>
  </widget>
  <widget class="QLabel" name="lablebrowser">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>601</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/Resources/shuimo3.png);
font: 20pt &quot;隶书&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>drug32</zorder>
  <zorder>drug22</zorder>
  <zorder>drug12</zorder>
  <zorder>qinglongjian</zorder>
  <zorder>zhengyangjian</zorder>
  <zorder>btnzhuqueguan</zorder>
  <zorder>btndrug3</zorder>
  <zorder>btndrug2</zorder>
  <zorder>btndrug1</zorder>
  <zorder>lablemoney</zorder>
  <zorder>money_2</zorder>
  <zorder>btnzhengyangguan</zorder>
  <zorder>btnbaihuxue</zorder>
  <zorder>btnzhengyangxue</zorder>
  <zorder>btnxuanwujia</zorder>
  <zorder>btnhuxinjing</zorder>
  <zorder>lablebrowser</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Menu.qrc"/>
 </resources>
 <connections/>
</ui>
